src/QueryEditorForm.tsx

```diff
@@ -45,40 +60,49 @@ export function QueryEditorForm(props: Props) {
 
   const templateVariables = props.datasource.getVariables();
 
-  const fetchRegions = () =>
-    props.datasource.getRegions().then((regions) => appendTemplateVariables(templateVariables, regions));
-  const fetchCatalogs = () =>
-    props.datasource
+  const fetchRegions = async () =>
+    await props.datasource
+      .getRegions()
+      .then((regions) => appendTemplateVariables(templateVariables, regions))
+      .finally(() => setNeedsUpdate(true));
```

Comment:
Updated to be async / await functions to match redshift, but doesn't impact functionality / fix at all